plugin/src/main/kotlin/org/jlleitschuh/gradle/ktlint/worker/KtLintWorkAction.kt

```diff
@@ -156,6 +156,15 @@ abstract class KtLintWorkAction : WorkAction<KtLintWorkAction.KtLintWorkParamete
         if (disabledRules.isNotEmpty()) {
             userData["disabled_rules"] = disabledRules.joinToString(separator = ",")
         }
+        if (SemVer.parse(parameters.ktLintVersion.get()) < SemVer(0, 48, 0)) {
+            if (disabledRules.isNotEmpty()) {
+                userData["disabled_rules"] = disabledRules.joinToString(separator = ",")
+            }
+        } else {
+            disabledRules.forEach {
+                userData["ktlint_standard_$it"] = "disabled"
```

Comment:
this is not safe, because not all rules are in the standard set.
there is really no way to know, just from the rule name, which set it is in.
supporting this would also require that plugin users include the full name with ruleset.
because of this, we are now recommending to just configure disabled rules using .editorconfig

Reply:
I understood the policy of abolishing disabledRules and setting only with .editorconfig.